src/connection.rs

```diff
@@ -347,8 +355,19 @@ async fn listen_on_uni_streams(
             }
         }
     } {
+        let mut break_ = false;
```

Comment:
Yup, it's exactly to avoid a keyword. I could have gone for something more creative but ended up with this 😄 I think a "raw" identifier would work as well (`r#break`) but I find them more awkward to read.

Not precious about this though, if we have a convention on it already.